miden-tx/src/host/tx_authenticator.rs

```diff
@@ -167,7 +120,7 @@ impl<R: Rng> TransactionAuthenticator for BasicAuthenticator<R> {
 /// Will return an error if either:
 /// - The secret key is malformed due to either incorrect length or failed decoding.
 /// - The signature generation failed.
-fn get_falcon_signature<R: Rng>(
+pub fn get_falcon_signature<R: Rng>(
```

Comment:
Also made this change so that it can be used for other authenticator implementations.